fdbclient/include/fdbclient/SystemData.h

```diff
@@ -539,6 +541,8 @@ extern const KeyRef backupLatestVersionsPrefix;
 // Key range reserved by backup agent to storing mutations
 extern const KeyRangeRef backupLogKeys;
 extern const KeyRangeRef applyLogKeys;
+// Returns true if m is a blog or alog mutation
```

Comment:
nit: `blog` should be read as `b-log` :)

Reply:
I think we refer to it as `blog` in other places in the code so would prefer to keep it as is

Reply:
Interesting. Sounds good to me, but certainly not a `readable` name, especially when `blog` is quite well known word :)

Reply:
I changed the comment to be a bit more descriptive so it should make it clearer